### ISSUE TITLE
Improve Firefox Android Extension (and get officially endorsed as a default option)

### ISSUE THREAD

**Issue by MintMain21** [einaregilsson/Redirector#333]
As I alluded to in #332, we've gotten a proper announcement from Firefox Android regarding a loose roadmap for 2023. The endgoal seems to be offering proper Manifest 3 support in Android before widening official support to virtually every extension available. https://blog.mozilla.org/addons/2022/12/15/new-extensions-available-now-on-firefox-for-android-nightly/.

From my experience, Redirector is perfectly functional in the latest versions of Firefox Android, with the exception of editing Redirects from the extension's settings page (the text dissapears). There is a workaround in the form of exporting the rules and editing them in a text editor, but this I fear is not user friendly enough to make Redirector approachable for Default Extension Status. 

TLDR, if we want to get Redirector properly endorsed by Firefox Android as a default option without using a Firefox Nightly Add-on Collection Hack, this issue would need to be resolved, whether that means migrating Redirector to Manifest 3 or not.

**Comment by igorzhilin** [comment]
Good idea. Fyi, I use Redirector with Iceraven on Android. It's a pretty decent fork of FF that is not tied to Mozilla and supports more extensions than FF mobile.
